src/main/java/mops/applicationServices/DatePollCreateService.java

```diff
@@ -16,118 +21,108 @@
-     * @param datePollMetaInf Meta-Informationen: Titel, Ort, Beschreibund.
+     * @param datePollMetaInfDto     Ein Dto Objekt welches die Meta-Informationen.
      * @return Lombok-Builder DatePoll Objekt.
      */
-    public DatePollBuilderAndView addDatePollMetaInf(final DatePollBuilderAndView datePollBuilderAndView, final DatePollMetaInf datePollMetaInf) {
+    public void addDatePollMetaInf(final DatePollBuilderAndView datePollBuilderAndView, final DatePollMetaInfDto datePollMetaInfDto) {
         DatePollBuilder builder = datePollBuilderAndView.getBuilder();
-        builder.datePollMetaInf(new DatePollMetaInf()); //Title Location Description
-        return datePollBuilderAndView.updateState(builder);
+        builder.datePollMetaInfDto(datePollMetaInfDto);
+        datePollBuilderAndView.setMetaInfDto(datePollMetaInfDto);
     }
 
 
-
     /**
      * Vierte Instanz zur Erstellung der Terminfindung.
      * Liste der Datum-Eintraege an denen der Termin stattfinden kann.
+     *
      * @param datePollBuilderAndView Das Lombok-Builder Objekt aus initializeDatePoll.
-     * @param datePollOptions Alle Datums-Eintraege des User creator.
+     * @param datePollOptionDtos     Alle Datums-Eintraege des User creator.
      * @return Lombok-Builder DatePoll Objekt.
      */
-    public DatePollBuilderAndView initDatePollOptionList(final DatePollBuilderAndView datePollBuilderAndView, final List<DatePollOption> datePollOptions) {
+    public void initDatePollOptionList(final DatePollBuilderAndView datePollBuilderAndView, final List<DatePollOptionDto> datePollOptionDtos) {
         DatePollBuilder builder = datePollBuilderAndView.getBuilder();
-        builder.datePollOptionList(datePollOptions);
-        return datePollBuilderAndView.updateState(builder);
+        builder.datePollOptionDtos(datePollOptionDtos);
+        datePollBuilderAndView.setDatePollOptionDtos(datePollOptionDtos);
     }
 
-
-
     /**
      * Handelt es sich um eine oeffentliche oder geschlossene Abstimmung?
+     *
      * @param datePollBuilderAndView
      * @param openDatePoll
      * @return
      */
-    public DatePollBuilderAndView openOrClosedPoll(final DatePollBuilderAndView datePollBuilderAndView, final boolean openDatePoll) {
+    public void openOrClosedPoll(final DatePollBuilderAndView datePollBuilderAndView, final boolean openDatePoll) {
         DatePollBuilder builder = datePollBuilderAndView.getBuilder();
-        DatePollConfig oldConfig = datePollBuilderAndView.getUnfinishedConfig();
-        DatePollConfig newConfig = oldConfig.withSingleChoiceDatePoll(openDatePoll);
-        builder.datePollConfig(newConfig);
-        return datePollBuilderAndView.updateState(builder);
+        DatePollConfigDto oldConfig = datePollBuilderAndView.getConfigDto();
+        DatePollConfigDto newConfig = oldConfig.withSingleChoiceDatePoll(openDatePoll);
+        builder.datePollConfigDto(newConfig);
+        datePollBuilderAndView.setConfigDto(newConfig);
```

Comment:
Kann sicherlich sinnvoll refactored werden in eine updateConfig-Methode, ähnlicher/gleicher Code kommt noch dreimal vor, daher besonders sinnvoll (code-duplication vermeidung)